config/initializers/rufus.rb

```diff
@@ -0,0 +1,17 @@
+require 'rufus-scheduler'
+# Let's use the rufus-scheduler singleton
+s = Rufus::Scheduler.singleton
+# Slack notifications linked to webhook that is directed to the channel random with the name overdue notifier
+SLACK_NOTIFIER1 = Slack::Notifier.new(
+	"https://hooks.slack.com/services/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", channel: "#random", username: "OVERDUE NOTIFIER")
```

Comment:
The webhook needs to be in the environmental variables instead of the hard coded